lib/query-interface.js

```diff
@@ -537,7 +537,9 @@ class QueryInterface {
     options = _.clone(options) || {};
     options.type = QueryTypes.INSERT;
     const sql = this.QueryGenerator.bulkInsertQuery(tableName, records, options, attributes);
-    return this.sequelize.query(sql, options);
+    return this.sequelize.query(sql, options).then(results => {
+      return _.head(results);
```

Comment:
@janmeier With ES6 and arrow functions all these bluebird constructs can be expressed much more concise without having to know the bluebird API. I think where possible we should just use an arrow function expression, it is easier to understand